backend-deploy-cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
@MakarkinSAkvelon @ruslan-ikhsan  I think this file should not be in the root folder of beam repo. Can we have it somewhere near playground code?